File: test/logicalOps.pl
#!/usr/local/bin/perl -w

$a=1;
$b=0;

if($a==$a && $b==$b){
        print "True\n";
}
if($a==$a || $b==$a){
	print "True\n";
}
if(!($a==$b || $b==$a)){
        print "True\n";
}

File: test/logicalOps.py
#!/usr/bin/python2.7 -u
import sys
import re

a=1
b=0

if ( a== a and  b== b): 
#variable interpolation
        sys.stdout.write("True\n".format(**locals()))
if ( a== a or  b== a): 
#variable interpolation
	sys.stdout.write("True\n".format(**locals()))
if (not( a== b or  b== a)): 
#variable interpolation
        sys.stdout.write("True\n".format(**locals()))

